kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/lemmas.k

```diff
@@ -239,4 +239,7 @@ module LEMMAS-HASKELL [symbolic]
 
     rule (notBool (A andBool B)) andBool A => (notBool B) andBool A [simplification]
 
+    rule [notBool-or]:  notBool ( A  orBool B ) => ( notBool A ) andBool ( notBool B ) [simplification(60)]
+    rule [notBool-and]: notBool ( A andBool B ) => ( notBool A )  orBool ( notBool B ) [simplification(60)]
+
```

Comment:
Set a later priority for these rules so they don't conflict with the preceding ones.